core/src/test/java/nl/nn/adapterframework/pipes/ExceptionPipeTest.java

```diff
@@ -0,0 +1,54 @@
+package nl.nn.adapterframework.pipes;
+
+import nl.nn.adapterframework.core.PipeRunException;
+import org.junit.Test;
+import org.junit.Before; 
+import org.junit.After;
+
+import java.nio.channels.Pipe;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * ExceptionPipe Tester.
+ *
+ * @author <Sina Sen>
+ * @version 1.0
+ * @since <pre>Mar 5, 2020</pre>
```

Comment:
> We don't really use the since tag. When we do use these, we use the version number and not the date.
> For all these tests you can remove them, as they don't add any additional value to the file(s).

okay! I used them to index what  I did on which day, I'm going to remove them.